auth/login.html

```diff
@@ -1,15 +1,21 @@
 {{define "content"}}
-	<h2>Login</h2>
-	<form method="post">
-		<div class="form-group">
-			<label for="username">User</label>
-			<input class="form-control" type="text" id="username" name="username" />
+		<div class="container">
+			{{template "login-head" .}}
+			<div class="login-container col-md-4">
+				<h3>Log in</h3>	
+				<form>
+					<input type="text" placeholder="Email" class="full-width">	
```

Comment:
Missing `.form-group` it's still good to have labels even if you don't show them, for accessibility reasons.